lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -66,8 +76,10 @@ class ElavonGateway < Gateway
       # * <tt>:password</tt> -- PIN
       # * <tt>:user</tt> -- Specify a subuser of the account (optional)
       # * <tt>:test => +true+ or +false+</tt> -- Force test transactions
+      # * <tt>:custom_fields_array => custom fields configured on specific Elavon account</tt> (optional)
       def initialize(options = {})
         requires!(options, :login, :password)
+        @custom_fields_array = options[:custom_fields_array] || [:customer_number]
```

Comment:
no problem changing custom_fields_array to custom_fields.  Can you clarify what you mean by changing default to constant?  Do you mean `customer_number`?